includes/templates/template_default/templates/tpl_account_history_info_default.php

```diff
@@ -111,7 +111,19 @@
     <tr>
         <td><?php echo zen_date_short($statuses['date_added']); ?></td>
         <td><?php echo $statuses['orders_status_name']; ?></td>
-        <td><?php echo (empty($statuses['comments']) ? '&nbsp;' : nl2br(zen_output_string_protected($statuses['comments']))); ?></td>
+        <td>
+<?php 
+    if (empty($statuses['comments'])) {
+echo '&nbsp;'; 
```

Comment:
I know we often stick a non-break-space into an empty `<td>` in order for it to display "properly" because of html4 challenges. 

Is this still needed in more modern browsers and html5? If not, let's simplify this by removing it here.